atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -185,66 +141,62 @@ const CreateCrashRecord = ({ client }) => {
               index++
             ) {
               personObjects = personObjects.concat(`{
-        crash_id: ${tempId},
         unit_nbr: ${unitNumber},
         prsn_injry_sev_id: 1,
+        cris_schema_version: "2023",
+        is_primary_person: true,
+        updated_by: "${userEmail}",
+        created_by: "${userEmail}"
       }`);
             }
-          });
 
-          personObjects = personObjects.concat("]");
-          unitObjects = unitObjects.concat("]");
+            unitObjects = unitObjects.concat(`{
+              unit_nbr: ${unitNumber},
+            unit_desc_id: ${Number(unit.unit_desc_id)},
+            cris_schema_version: "2023",
+            updated_by: "${userEmail}",
+            created_by: "${userEmail}",
+            people_cris: {
+              data: [
+                ${personObjects}
+              ]
+            }
+          }`);
```

Comment:
this works fine, but i'd prefer to to see the entire crash record object be included as a single `crashes_cris_insert_input` graphQL variable. it's just that the string-formatting tends to be error-prone and a bit difficult to maintain.